tests/std/tests/P0896R4_ranges_alg_is_permutation/test.cpp

```diff
@@ -88,6 +88,52 @@ struct instantiator {
             assert(!ranges::is_permutation(ranges::begin(r1), ranges::end(r1), ranges::begin(r2), ranges::end(r2),
                 ranges::equal_to{}, get_first, identity{}));
         }
+        {
+            int arr1[] = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9};
+            int arr2[] = {9, 8, 7, 6, 5, 4, 3, 2, 1, 0};
+            const Fwd2 r1{arr1};

```

Comment:
It appears that these newly added test cases should say `const Fwd1 r1` to compare potentially different range types.

Reply:
The problem is that `Fwd1` is not a range of `int`s.

Reply:
Ah, ok!